src/Configuration/Profile.cs

```diff
@@ -344,6 +345,12 @@ private void SaveGumps(string path)
                     }
                 }
 
+                // Save NameOverHeadHandlerGump's last position even if it's not opened on save unless it's at default position
+                if (!gumps.Any(g => g.GumpType == GumpType.NameOverHeadHandler) && NameOverHeadHandlerGump.LastPosition.HasValue && NameOverHeadHandlerGump.LastPosition != new Point(100, 100))
+                {
+                    var nameOverHeadHandlerGump = new NameOverHeadHandlerGump();
+                    gumps.AddLast(nameOverHeadHandlerGump);
+                }
```

Comment:
Yeah that could work, so instead of spawning/despawning the gump every time the user presses ctrl+shift, we just create it once and disable it? I can look into changing the OverHeadGumpManager so it works like that

Reply:
Exactly, a single persistent gump that is visible when pressing cltr shift

Reply:
Okay, I pushed another commit that is now only instantiating one gump and setting disabled and visible to true/false based on open/close. The initial instantiation still only happens when ctrl+shift is pressed for the first time to save some resources if a player never uses it.